### ISSUE TITLE
Update to Outlook weekday recurrence fails - 2-way sync changing interval 0 to 1

### ISSUE THREAD

**Issue by TJ22** [phw198/OutlookGoogleCalendarSync#398]
[OGcalsync.log.txt](https://github.com/phw198/OutlookGoogleCalendarSync/files/1505385/OGcalsync.log.txt)

**OGCS Version**: v2.6.4.0 
**Installed or Portable**: Installed

**Problem Description**  
\Error with each sync. 
WARNING: Updated appointment failed to save.
Mitglied nicht gefunden. (Ausnahme von HRESULT: 0x80020003 (DISP_E_MEMBERNOTFOUND))

**Comment by phw198** [comment]
So the section of log file is:
```
2017-11-27 11:00:18,765 DEBUG  16 OutlookGoogleCalendarSync.OutlookOgcs.Calendar [0] -  Processing >> 09.10.2017 08:45 (R) => "Morgenrunde"
2017-11-27 11:00:18,796 DEBUG  16 OutlookGoogleCalendarSync.Recurrence [0] -  Converting RRULE:FREQ=WEEKLY;UNTIL=20171231T074500Z;BYDAY=MO,TU,WE,TH,FR
2017-11-27 11:00:18,827 DEBUG  16 OutlookGoogleCalendarSync.MainForm [0] -  09.10.2017 08:45 (R) => "Morgenrunde"
Recurrence Interval: 0 => 1
```
...OGCS wants to update the Outlook recurrence interval from 0 to 1. This seems entirely reasonable, in fact an interval of 0 is not a valid recurrence pattern, so it feels to me that your Outlook appointment series has got a problem.

Could you open the `09.10.2017 08:45 => "Morgenrunde"` series and see what the "Recur every x week(s) on" value is? If you simply click "OK" and then `Ctrl+S` to save the appointment, does Outlook complain?

Try changing the recurrence to "1" anyway and see if this works around the problem.

**Comment by TJ22** [comment]
It is a daily event, but only working days.
Outlook does not complain.

**Comment by phw198** [comment]
So in Outlook is it a **Daily** recurrence **Every weekday**?  
Because in Google it's a **Weekly** recurrence for **Mon,Tue,Wed,Thu,Fri**.

What was the order of events?
1. Created in Outlook
1. Synced by OGCS to Google
1. Updated in Google
1. Synced to Outlook and fails

**Comment by TJ22** [comment]
Right, daily recurrence every weekday.
1. Created in Outlook
2. Synced by OGCS to Google.
That's it! No Update in Google!

**Comment by phw198** [comment]
:package: Please could you see if this is resolved with [v2.6.4.1.zip](https://github.com/phw198/OutlookGoogleCalendarSync/files/1525256/v2.6.4.1.zip) ([see instructions for applying](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/Reporting-Problems#applying-a-hotfix)).

It needs to be applied on top of Alpha v2.6.4. 

Unfortunately it looks like this only affects certain versions of Outlook (I see you're on Outlook 2007), as I couldn't actually replicate the problem. Anyway, let me know how it goes.

**Comment by TJ22** [comment]
Okay, now it works. Awesome!
Thank you very much
